pkgs/development/mobile/androidenv/androidsdk.nix

```diff
@@ -73,11 +73,12 @@ stdenv.mkDerivation rec {
         --prefix PATH : ${jdk}/bin \
         --prefix LD_LIBRARY_PATH : ${stdenv.lib.makeLibraryPath [ glib gtk2 libXtst ]}
 
-      # The emulators need additional libraries, which are dynamically loaded => let's wrap them
 
       ${stdenv.lib.optionalString (stdenv.hostPlatform.system == "x86_64-linux") ''
         for i in emulator emulator-check
         do
+            patchelf --set-interpreter ${stdenv.cc.libc.out}/lib/ld-linux-x86-64.so.2 $i
```

Comment:
There is this pull request that also extends the rpath: https://github.com/NixOS/nixpkgs/pull/49515/files

Reply:
Hmm, I was able to run the binaries just by changing the interpreter, but maybe there were bugs I wasn't hitting.

Reply:
After thinking about it a bit I think it might be also not necessary because the wrapper already puts the same shared library directory with LD_LIBRARY_PATH into the scope.